src/commands/infusion.ts

```diff
@@ -0,0 +1,105 @@
+import { find } from "lodash";
+import { gods } from "../data/gods/all";
+import { AIR, BoonElement, COSMIC, EARTH, FIRE, WATER } from "../data/gods/elements";
+import { InfusionBoon, isInfusion } from "../data/gods/god";
+import { Command } from "./command";
+
+const allElements = [EARTH, WATER, AIR, FIRE, COSMIC];
+const allElementNames = allElements.join("|");
+const infusionCommandExpression = RegExp(`^(infusions?) ?(${allElementNames})?$`, "i");
+
+type InfusionRecord = {
+  boon: string;
+  god: string;
+};
+const elementInfusionMap = gods.reduce<Record<BoonElement, InfusionRecord[]>>(
+  (hash, god) => {
+    // Assumes each god has no more than one infusion, which is currently true.
+    let infusionBoon = find(god.abilities, (boon, _) =>
+      isInfusion(boon)
+    ) as InfusionBoon;
+    if (!infusionBoon || !infusionBoon.requiredElements) {
+      return hash;
+    }
```

Comment:
Lodash (which is where `find` comes from) has a bunch of really nice utility methods, but unfortunately it's typed really poorly and its failing you here. If you avoid it using native methods it'll clean this up:
```
const infusionBoon = Object.values(god.abilities).find(isInfusionBoon)
// Resulting type is InfusionBoon | undefined
```